src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenFunctionPointersTests.cs

```diff
@@ -51,12 +57,14 @@ private CSharpCompilation CreateCompilationWithFunctionPointers(string source, I
         [InlineData("stdcall", CallingConvention.Standard)]
         internal void CallingConventions(string conventionString, CallingConvention expectedConvention)
         {
-            var comp = CompileAndVerifyFunctionPointers($@"
+            var verifier = CompileAndVerifyFunctionPointers($@"
 class C
 {{
     public unsafe delegate* {conventionString}<string, int> M() => throw null;
 }}", symbolValidator: symbolValidator);
 
+            symbolValidator((ModuleSymbol)((Symbols.PublicModel.ModuleSymbol)verifier.Compilation.SourceModule).UnderlyingSymbol);
```

Comment:
>(ModuleSymbol)((Symbols.PublicModel.ModuleSymbol)verifier.Compilation.SourceModule).UnderlyingSymbol [](start = 28, length = 100)

```((CSharpCompilation)verifier.Compilation).SourceModule```?

Reply:
And below as well.

---
In reply to: [413317301](https://github.com/dotnet/roslyn/pull/43355#discussion_r413317301) [](ancestors = 413317301)